src/common/workspace-store.ts

```diff
@@ -51,6 +51,10 @@ export class WorkspaceStore extends BaseStore<WorkspaceStoreModel> {
 
   @action
   setActive(id = WorkspaceStore.defaultId) {
+    if (!this.getById(id)) {
+      throw new Error(`workspace ${id} doesn't exist`);
```

Comment:
Maybe change first letter to capital one?

Reply:
Not that I am a big fan of `golang` but I do like their idea that error strings should start lowercase, so that when joined with other strings there do not end up random capitalized letters in the middle of sentences.